openfecwebapp/templates/datatable.html

```diff
@@ -48,7 +47,14 @@
 {% if office %}
 <script>
 var context = {
-  office: '{{ office }}'
+  office: '{{ office }}',
+};
+</script>
+{% endif %}
+{% if form_type %}
+<script>
+var context = {
+  formType: '{{ form_type }}',
```

Comment:
So yeah, that's the idea: the `office` variable is only being passed through on candidate office pages, and the `form_type` variable is only being passed through on reports pages. But that's a good point: it's definitely brittle and liable to cause problems if we ever end up trying to pass both. I'll think of a better way to do it.